left_column.md

```diff
@@ -75,6 +75,7 @@ The focus of this workshop is on:
 
 - 16:30 - 17:00
   Group work: Lesson design
+  [Group work: Lesson design](https://coderefinery.github.io/instructor-training/lesson-design/#practice-backwards-design)
```

Comment:
Is this a duplicate / should top one be removed?

Reply:
Oh yes, duplicate.